src/inject/planner_helper.js

```diff
@@ -86,6 +86,8 @@ PlannerHelper.prototype.makeViewDataButton = function (teacher, course) {
     button.data('teacher', teacher);
     button.on('click', function () {
         plannerHelper.reloadData($(this).data('teacher'), $(this).data('course'));
+        var elem = document.getElementById('planner-helper');
```

Comment:
Actually, I think this should be making use of instance variables. If you look at [the code](https://github.com/krau612/planner-helper/blob/216e1a099005f9bbcd0e6be422f0374bea887ba7/src/inject/planner_helper.js#L11) you'll notice that there is a variable `this.element`